.drone.star

```diff
@@ -622,26 +622,39 @@ def website(ctx):
     },
     'steps': [
       {
-        'name': 'generate',
-        'image': 'webhippie/hugo:latest',
-        'pull': 'always',
+        'name': 'prepare',
+        'image': 'owncloud/alpine:latest',
         'commands': [
-          'make docs',
+          'apk add make',
+          'make docs-copy'
         ],
       },
       {
-        'name': 'publish',
-        'image': 'plugins/gh-pages:1',
-        'pull': 'always',
+        'name': 'test',
+        'image': 'klakegg/hugo:0.59.1-ext-alpine',
```

Comment:
Cool, now using some random guy docker image?

Reply:
suggestions?

Reply:
Switched back to webhippie